bigtable/client/detail/bulk_mutator.cc

```diff
@@ -48,10 +49,6 @@ BulkMutator::BulkMutator(std::string const &table_name,
                          });
     pending_annotations_.push_back(Annotations{index++, r, false});
   }
-
-  // Make a copy of the table name into the request proto, that way we do not
-  // have to initialize it ever again.
-  mutations_.set_table_name(table_name);
```

Comment:
I understand the reordering change above; I'm not sure why this line is being deleted though.

Reply:
This line has I effect.  That value is overwritten later visa `Swap()` call.

Sorry I cannot send a link, answering from my phone.

Reply:
Thanks, I see it in `PrepareForRequest()`.